CHANGES.rst

```diff
@@ -21,6 +21,15 @@ Release History
 2.2.1 (unreleased)
 ==================
 
+**API changes**
```

Comment:
In 2.2.0 + we're switching the changelog format to be Added/Removed/Changed/Fixed; removed probably makes sense right? Or do we want a deprecated section?